tox.ini

```diff
@@ -0,0 +1,38 @@
+[tox]
+requires =
+    tox>=4.2
+env_list =
+    py
+    lint
+
+[testenv]
+skip_install = true
+deps =
+    -r dev-requirements.txt
+commands =
+    {envpython} -m pytest \
+      tests/ \
+      --cov . \
+      --cov tests \
+      --cov-report html \
+      --cov-report term \
+      --cov-report xml \
+      {posargs}
+
+[testenv:lint]
+skip_install = true
+deps =
+    pre-commit
+pass_env =
+    PRE_COMMIT_COLOR
+commands =
+    pre-commit run --all-files --show-diff-on-failure
+
+[testenv:mypy]
+skip_install = true
+deps =
+    mypy==1.10
```

Comment:
Will dependabot update this?

Reply:
It won't, I'll move it to a requirements file.